### DISCUSSION TITLE
建议不要限制顶栏的位置

### DISCUSSION THREAD

**Discussion by zuoer96** [travellings-link/travellings#1579] [发展建议]
建议不要限制顶栏的位置，我不觉得强制用户占据一个菜单位置是什么好事。
相较之下，十年之约只对用户的内容做限制。
我觉得开往，也应该关注的是用户的内容，至于用户是否放置，应该交给用户选择。

**Comment by linlinzzo** [comment]
目前，在实践的过程当中，我们只是希望将徽标放在显眼的位置。如果成员的主题，没有办法配置的话，那我们也可以接受他放在底部。

**Comment by linlinzzo** [comment]
@Big-Cake-jpg @baipin 你们怎么看？

**Reply by Big-Cake-jpg** [reply]
十年之约只关心内容是因为其主要功能不是类似开往的这种友链接力。
强制用户给开往留一个菜单确实不是什么好事，但是如果用户把外链放在了底栏可能会友链接力的连续性？
总之，我的想法是不限制开往外链的位置，只要是个全局的、明显的地方就行
